packages/compass-import-export/src/components/progress-bar/progress-bar.jsx

```diff
@@ -133,13 +133,13 @@ class ProgressBar extends PureComponent {
 
     return (
       <div className={style('chart-wrapper')}>
-        {docsTotal && (
+        {!isNaN(docsTotal) && (
```

Comment:
Drive-by to fix `0` rendering for a split second (value needs to be false, null, or undefined for react not to render it)